.github/workflows/ci.yml

```diff
@@ -36,5 +36,10 @@ jobs:
     - name: Install dependencies
       run: pip install -r requirements_dev.txt
 
+    # TODO remove once released
+    - name: Install custom libpetab-python
+      run: |
+        pip install --upgrade --upgrade-strategy eager -e git+https://github.com/PEtab-dev/libpetab-python.git@develop
```

Comment:
I think this won't work, the line must instead be added to the tox.ini:testenv:base commands list, as tox uses an own env

Reply:
Thanks, will opt for putting it in `setup.py` for now, then update to next libpetab-python version, when available.